include/meson.build

```diff
@@ -19,7 +21,7 @@ public_headers = [
    'taskolib/taskolib.h',
    'taskolib/time_types.h',
    'taskolib/Timeout.h',
-   'taskolib/VariableName.h',
+   'taskolib/VariableName.h'
```

Comment:
Meson allows you to put an extra comma after the last entry in a list. This way the diffs look a little nicer if you add a line later on. (No need to change this, though.)